java/pgv-java-stub/src/main/java/io/envoyproxy/pgv/StringValidation.java

```diff
@@ -93,6 +93,14 @@ public static void email(String field, String value) throws ValidationException
         }
     }
 
+    public static void address(String field, String value) throws ValidationException {
+        try {
+            hostName(field, value);
+        } catch(ValidationException expected) {
+            ip(field, value);
+        }
```

Comment:
When `value` is neither a host name nor an IP address, the resulting validation exception will say `XXX should be a valid ip address` because the host name error is lost. 

The response should be something like `XXX should be a valid host name or ip address`.

Reply:
I opted to use: "host" here instead because that's what validation below says. However, if you'd like I can change them all to "hostname".

Thanks for catching this!

Reply:
The specific wording isn't too important. The underlying bug is that the IP validator's exception is what will be bubbled up to the client if this check fails, which will result in a misleading response.